pilot/pkg/proxy/envoy/v2/eds.go

```diff
@@ -374,30 +376,6 @@ func (s *DiscoveryServer) updateCluster(push *model.PushContext, clusterName str
 // SvcUpdate is a callback from service discovery when service info changes.
 func (s *DiscoveryServer) SvcUpdate(cluster, hostname string, ports map[string]uint32, _ map[uint32]string) {
 	inboundServiceUpdates.Increment()
-	pc := s.globalPushContext()
```

Comment:
What happened with this code ? No longer needed to track by port ?

Reply:
If that is the case - maybe remove the whole callback/interface.

Reply:
The intent was because the port map was replaced by `ServiceByHostname.Port`. maybe we need to update ServiceByHostname here instead.. not sure how it worked before -- when we called SvcUpdate ServiceByHostname would become stale?